src/lib/serviceRoutines/postRegisterContext.cpp

```diff
@@ -58,6 +58,28 @@ std::string postRegisterContext
 {
   RegisterContextResponse  rcr;
   std::string              answer;
+  RegisterContextRequest*  rcrP = &parseDataP->rcr.res;
+
+  //
+  // Check for isPatterns in request
+  //
+  for (unsigned int crIx = 0; crIx < rcrP->contextRegistrationVector.size(); ++crIx)
+  {
+    ContextRegistration* crP = rcrP->contextRegistrationVector[crIx];
+
+    for (unsigned int eIx = 0; eIx < crP->entityIdVector.size(); ++eIx)
+    {
+      if (isTrue(crP->entityIdVector[eIx]->isPattern))
+      {
+        std::string  details = "isPattern set to true for a registration";
```

Comment:
BTW, maybe the doc at https://fiware-orion.readthedocs.io/en/develop/user/context_providers/index.html should mention the limitation at the end in the "additional comments bullet list".